mindmeld/components/nlp.py

```diff
@@ -684,22 +706,21 @@ def process(
                 applying the full hierarchy of natural language processing models to the input \
                 query.
         """
+        # TODO: Deprecate language argument
+        del language
```

Comment:
doc string should indicate param is ignored

Reply:
done